app/models/case_log.rb

```diff
@@ -520,7 +524,7 @@ def tenancylength_optional?
 
   def set_derived_fields!
     # TODO: Remove once we support supported housing logs
-    self.needstype = 1 unless needstype
+    self.needstype = 1 unless needs_question_enabled?
```

Comment:
I think I'd use the feature toggle class directly here rather than a wrapper method so it's more obvious this *is* a feature flag and not application logic.

Reply:
I did it this way since the depends_on currently relies on the methods in question being part of the case log model

Reply:
I wasn't quite sure how to do this without modifying the depends on method

Reply:
As in, I would probably just do:

```ruby
self.needstype = 1 unless FeatureToggle.needs_question_enabled?
```

Reply:
Ah whoops I misread where this comment was made - ignore me 🤦